lib/matplotlib/backend_bases.py

```diff
@@ -1425,6 +1426,12 @@ def _update_enter_leave(self):
         LocationEvent.lastevent = self
 
 
+class MouseButton(IntEnum):
```

Comment:
could we change this to an `Enum`  and add

~~~
UP = 'up'
DOWN = 'down'
~~~

Reply:
Again, this will break uses like `event.button == 1`.
(I guess we could override `__eq__` if you really want to...).